tests/runtime-trace-tests/cases/global_array_len.py

```diff
@@ -0,0 +1,9 @@
+choices = ["rock", "paper", "scissors"]
+
+x = len(choices)
+
+def next_choice(forward):
```

Comment:
Why is this in the trace test if it never gets called?

Reply:
Trace tests also test that compilation doesn't error py->ts and back to py.
But yeah, I might as well make a trace here